### PR TITLE
feat | fix execution alias chat on actual status

### PR DESCRIPTION
**Pull request by Baghdasarov**
**Please check if the PR fulfills these requirements**:

-   [X] The commit message follows [our code of conduct](https://github.com/codemod-com/codemod-registry/blob/main/CODE_OF_CONDUCT.md) and [contributing guidelines](https://github.com/codemod-com/codemod-registry/blob/main/CONTRIBUTING.md)
-   [X] Tests for the changes have been added (for bug fixes/features)
-   [X] Docs have been added / updated (for bug fixes / features)

**Please include the following information in your pull request**:

-   **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

-   **What is the current behavior?** (You can also link to an open issue here)

-   **What is the new behavior (if this is a feature change)?**

-   **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

-   **Any other information (if needed)**:

### CODE REVIEW

**Comment by mohab-sameh** [comment]
@all-contributors add @Baghdasarov for code and bug fixes

File: apps/studio/src/store/slices/CFS/alias.ts [hunk 1]
```diff
@@ -10,7 +10,24 @@ type AliasName =
...
+	let codemodExecutionObj = null;
+
+	if (codemodExecutionError !== null) {
+		codemodExecutionObj = {
+			value: codemodExecutionError ?? "",
+			updatedAt: rangesUpdatedAt,
+		};
+	}
+
+	return codemodExecutionObj;
```
Comment (hbjORbj) [inline apps/studio/src/store/slices/CFS/alias.ts:29]:
```suggestion
	if (codemodExecutionError === null) {
	        return null;
	}

	return  {
			value: codemodExecutionError,
			updatedAt: rangesUpdatedAt,
		};
```

File: apps/studio/src/store/slices/CFS/alias.ts [hunk 2]
```diff
@@ -66,10 +83,10 @@ const getAliases = (state: RootState): Aliases => {
 					updatedAt: afterRangeUpdatedAt,
 			  }
 			: null,
-		$EXECUTION_ERROR: {
-			value: codemodExecutionError ?? "",
-			updatedAt: rangesUpdatedAt,
-		},
+		$EXECUTION_ERROR: generateCodemodExecutionObject(
```
Comment (hbjORbj) [inline apps/studio/src/store/slices/CFS/alias.ts:86]:
```suggestion
		$EXECUTION_ERROR: generateCodemodExecutionErrorObject(
```

File: apps/studio/src/store/slices/CFS/alias.ts [hunk 3]
```diff
@@ -10,7 +10,24 @@ type AliasName =
 	| "$HIGHLIGHTED_IN_AFTER"
 	| "$EXECUTION_ERROR";
 
-type Aliases = Record<AliasName, { value: string; updatedAt: number } | null>;
+type AliasObj = { value: string; updatedAt: number };
+type Aliases = Record<AliasName, AliasObj | null>;
+
+const generateCodemodExecutionObject = (
```
Comment (hbjORbj) [inline apps/studio/src/store/slices/CFS/alias.ts:16]:
```suggestion
const generateCodemodExecutionErrorObject = (
```

**Comment by hbjORbj** [comment]
@Baghdasarov Thanks for opening the PR. I left a few comments!

**Pull request closed without merging**